src/css/blocks/page.css

```diff
@@ -0,0 +1,91 @@
+.page {
+  position: relative;
+  z-index: 1;
+
+  padding-bottom: 10px;
+  padding-top: 10px;
+}
+
+.page__wrapper {
+  max-width: 50%;
+  text-align: center;
```

Comment:
I think this is generally not a good practice to put such specific styles into a generic component. It may not be the case for this page, but in general, you can end up overriding it more times than you would declare it separately. Furthermore, this makes other blocks a bit less self-contained, so it may violate BEM principles.